relayer/src/cli/common.rs

```diff
@@ -76,11 +76,29 @@ pub struct EthereumSignerArgs {
 }
 
 #[derive(Args)]
+pub struct EthereumSignerPathArgs {
+    #[clap(flatten)]
+    pub ethereum_args: EthereumArgs,
+
+    /// Private key for fee payer
+    #[arg(long = "eth-fee-payer-path", env = "ETH_FEE_PAYER_PATH")]
+    pub fee_payer_path: PathBuf,
```

Comment:
```suggestion
    #[arg(long, env)]
    pub eth_fee_payer_path: PathBuf,
```

and for other new arguments too